docs/docs/workflows/steps/params/README.md

```diff
@@ -0,0 +1,45 @@
+# Params
```

Comment:
Can you add front-matter at the top of this to disable the next / prev links at the bottom? Previous points to legacy actions. Feel free to revisit that logic altogether - I don't think it would matter if we removed them globally.

<img width="786" alt="Screen Shot 2022-04-05 at 10 23 46 AM" src="https://user-images.githubusercontent.com/242668/161814535-65f8d4b5-6bab-4f8a-b3a3-111bea6bf52a.png">

Reply:
@dylburger any issues with just removing that `/docs/workflow/actions/README.md` altogether?